.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -98,45 +131,3 @@ body:
       description: If you are aware of a workaround, please describe it below.
       placeholder: |
         e.g. There is an alternative way to access this setting in the sidebar, but it's not readily apparent.
-  - type: markdown
-    attributes:
-      value: |
-        ---
-        ## Additional context
-
-        Please provide whatever additional information you have available to you. If not, please scroll to the bottom and submit the issue.
-  - type: dropdown
-    id: os
```

Comment:
Did you intentionally drop the "Operating System" options, and the "Browser Version" option? Just checking to be sure.

Reply:
I did, yes, to match Calypso. Given that reporters seem to often ignore that option, and that more and more browsers have moved away from big version changes, I think it's safe to remove.